pkg/net/local/local.go

```diff
@@ -87,7 +89,8 @@ func (lp *localProvider) ConnectionManager() net.ConnectionManager {
 type localConnectionManager struct {
 	mutex sync.Mutex
 
-	peers map[string]*operator.PublicKey
+	peers         map[string]*operator.PublicKey
+	peersAddrInfo map[string]peer.AddrInfo
```

Comment:
It will always be empty, right? 

Then, `ConnectedPeersAddrInfo` will always return an empty slice even though `ConnectedPeers` will not be empty. 

Instead of adding a new mapping, I'd use `peers` in `ConnectedPeersAddrInfo` and returned some custom address, like `/local/$peerID`.

Reply:
Is this what you mean? [697609a](https://github.com/keep-network/keep-core/pull/3255/commits/697609a47913d9a2653081bb73c677d38c1d034c)